app/src/chat_engine.py

```diff
@@ -524,9 +534,6 @@ def on_message(
             f"System Prompt 1 (analyze_message) took {system_prompt_1_duration:.2f} seconds"
         )
 
-        if attributes.alert_message:
-            attributes.alert_message = f"**Policy update**: {attributes.alert_message}\n\nThe rest of this answer may be outdated."
-
```

Comment:
Moved `**Policy update**:` and `The rest of this answer may be outdated.` to the prompt so it can be translated.